### DISCUSSION TITLE
Android aar :  view.multiSampleAntiAliasingOptions not found.

### DISCUSSION THREAD

**Discussion by wingdi** [google/filament#4726] [Q&A]
code：

<img width="674" alt="截屏2021-10-13 上午8 43 26" src="https://user-images.githubusercontent.com/25787391/137047809-8e497157-4e8b-4f9d-a0c2-c3b67eec4a2f.png">

dependence:
<img width="598" alt="截屏2021-10-13 上午8 38 50" src="https://user-images.githubusercontent.com/25787391/137047526-81d3da34-dd6b-4ee7-a123-25f7cc79a26d.png">

seems like view.multiSampleAntiAliasingOptions didn't build into java code ？

**Comment by romainguy** [comment]
It's not part of 1.12.8, it will be in 1.12.9. The samples always build against tip of tree which is 1 week ahead of our releases.

**Reply by wingdi** [reply]
ok  thanks!
